### PR TITLE
Support BCH blockchain balance querying

### LINKED ISSUE #1028: Support BCH blockchain onchain balance querying
## Abstract

Add Bitcoin cash (BCH) blockchain support so that users can enter their BCH on chain addresses.

## Task

Do it. Probably will need to query something like blockchain.info for the prices. Will need to be careful to differentiate all over the codebase between bitcoin and bitcoin cash addresses as the sam address can exists in both chains. Need to make sure there is no mistaken assumptions anywhere.

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #1028

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/components/accounts/AccountBalanceTable.vue [hunk 1]
```diff
@@ -95,26 +95,24 @@
           <template v-if="!loopring">
             <account-asset-balances
               :title="$t('account_balance_table.assets')"
-              :assets="accountAssets(item.address)"
+              :assets="accountAssets(item.address).value"
```
Comment (kelsos) [inline frontend/app/src/components/accounts/AccountBalanceTable.vue:98]: is there not a better way to do this with composition other than calling `.value`?

**Review by kelsos — APPROVED** [review]
lgtm thanks

File: frontend/app/src/components/accounts/AccountBalanceTable.vue (hunk 1 above)
Comment (lukicenturi) [reply frontend/app/src/components/accounts/AccountBalanceTable.vue:98]: I can't find/think of other better ways.
The only thing that makes sense is to map the `accountAssets` for all data inside script, so inside the template, the function doesn't need to be called

Comment (kelsos) [reply frontend/app/src/components/accounts/AccountBalanceTable.vue]: ok, this is something for the future then.

**Pull request merged**